pkg/translator/faro/go.mod

```
@@ -3,23 +3,36 @@ module github.com/grafana/faro/pkg/translator/faro
 go 1.23.3
 
 require (
+	github.com/go-logfmt/logfmt v0.6.0
 	github.com/grafana/faro/pkg/go v0.0.0-20241118134905-713b31945e4d
+	github.com/stretchr/testify v1.9.0
 	go.opentelemetry.io/collector/pdata v1.20.0
```

```suggestion
	go.opentelemetry.io/collector/pdata v1.32.0
```

can we make sure all the otel deps are using the same version?

---

good point! updated all dependencies, the [latest version](https://github.com/open-telemetry/opentelemetry-collector/blob/fa4902638dc16dec2cd19c576e277e5107ebe0a8/versions.yaml) of the go.opentelemetry.io/collector/pdata is v1.21.0: